dockerfiles/docker-entrypoint.sh

```diff
@@ -101,18 +101,21 @@ function do_xhtml_validate() {
 
 function read_style() {
     slug_name=$1
-    if [[ ! -f $IO_BOOK/style ]]; then
-        die "Style not specified. File does not exist '$IO_BOOK/style'" # LCOV_EXCL_LINE
-    else
+    style_name=
```

Comment:
Is the style_name ok being defined w/o a value? Perhaps, it should default to an empty string?

Reply:
In bash scripts, that is the equivalent of initializing a variable as None/null. I could change it to an empty string if that's preferable. It should not impact functionality: empty strings are falsey.

Reply:
Well, I'm not sure what's best practice in bash. My personal preference is to set variables to None or an empty string if they need to be empty initially.

Reply:
If that's what Phil has done in previous scripts feel free to keep it the same.